.golangci.yml

```diff
@@ -11,3 +11,8 @@ linters:
     - errorlint
     - unconvert
     - unparam
+
+linters-settings:
+  govet:
```

Comment:
Shouldnt `govet` be added to the list of enabled linters above?

Reply:
Good question! It is included in the default set, so yes it is enabled. You can check it locally by running `golangci-lint linters` and making sure it is listed under "Enabled by your configuration linters".

Or, look at the doc at https://golangci-lint.run/usage/linters/ where govet is listed under "Enabled by default".

Basically, the "linters / enable" enables linters *in addition* to those enabled by default.